inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/utils/MySqlContainer.java

```diff
@@ -35,7 +35,7 @@ public class MySqlContainer extends JdbcDatabaseContainer {
     public static final String IMAGE = "mysql";
     public static final Integer MYSQL_PORT = 3306;
 
-    private static final String MY_CNF_CONFIG_OVERRIDE_PARAM_NAME = "MY_CNF";
+    private static final String MY_CNF_CONFIG_OVERRIDE_PARAM_NAME = "TC_MY_CNF";
```

Comment:
excuse me, any reasons  for change config name ?

Reply:
Mysql initialization will use this fixed name field  when the testcontainers run mysql docker contaner.

As https://java.testcontainers.org/modules/databases/mysql/ instruction